### DISCUSSION TITLE
Using ANTs for registration of 3D T1w MRI of the abdomen

### DISCUSSION THREAD

**Discussion by matthiasschabel** [ANTsX/ANTs#1338] [Q&A]
I'm wondering if anyone out there has a reasonable starting point for a robust ANTs-based pipeline for registering these types of images (with potential motion from respiration, bowel gas pockets, etc...)? The default settings for simple driver scripts like antsRegistrationSyNQuick don't seem to achieve adequate convergence. Before I plunge in, I'm hoping that someone else has already done something similar and might have suggestions on where to focus.

**Comment by gdevenyi** [comment]
I can suggest my more robust, https://github.com/CoBrALab/minc-toolkit-extras/blob/master/antsRegistration_affine_SyN.sh (which depends on https://github.com/CoBrALab/minc-toolkit-extras/blob/master/ants_generate_iterations.py) but its still generic and nothing special for the specific anatomical complications you're talking about.

**Comment by cookpa** [comment]
I haven't done lung registration myself but there's this from @ntustison : https://github.com/ntustison/antsCtLungRegistrationExample 

antsRegistrationSyNQuick.sh will attempt to do rigid and affine registration, which might not be a good idea if there's not a rigid / affine component to the transform.

Features present in one of the images (eg, air pockets) are a big problem for registration of all kinds. It might be necessary to add some manual segmentation, eg to mask them out.
